html-forms/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+  <html>
+
+  <head>
+    <title>HTML Forms</title>
+  </head>
+
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="" method="get">
+      <div>
+        <label for="email">Email</label>
+        <input id="email" type="text" name="email">
+      </div>
+      <div>
+        <label for="dropDownList">Why is coding cool?</label>
+      </div>
+      <div>
+        <select id="dropDownList" name="dropDownList">
+          <option value="">Select an option</option>
+          <option value="Because it's interesting">Because it's interesting</option>
+          <option value="Becuase it's fun">Becuase it's fun</option>
+          <option value="Because it's valuable">Because it's valuable</option>
+          <option value="Because it's powerful">Because it's powerful</option>
+        </select>
+      </div>
+      <div>
+       <p>Favorite Language (choose one)</p>
+      </div>
+      <div>
+        <label for="favoriteLanguage">
+          <input type="radio" name="favoriteLanguage" value="HTML">HTML
+          <input type="radio" name="favoriteLanguage" value="CSS">CSS
+          <input type="radio" name="favoriteLanguage" value="JavaScript">JavaScript
+        </label>
```

Comment:
This still only has one `label` for all three `input` elements. Each `input` element should have its own `label` element holding its label text.